### PR TITLE
editor/code: bump minimum version of VS Code to 1.78

### PR DESCRIPTION
**Pull request by davidbarsky**
Undoes https://github.com/rust-lang/rust-analyzer/pull/15333.

### CODE REVIEW

File: editors/code/package.json [hunk 1]
```diff
@@ -2178,4 +2178,4 @@
             }
         ]
     }
-}
+}
```
Comment (lnicola) [inline editors/code/package.json:2181]: Nit: can you leave this untouched?

**Comment by lnicola** [comment]
I think  you need to do `npm update @types/vscode`, otherwise you end up updating everything.

File: editors/code/package.json (hunk 1 above)
Comment (davidbarsky) [reply editors/code/package.json:2181]: whoops, sorry about that—i think my editor made this change.

**Comment by davidbarsky** [comment]
> I think you need to do `npm update @types/vscode`, otherwise you end up updating everything, including `tsc` to a version that's more strict when parsing its config file.

Cool, yeah—I think this PR was so large because I was futzing around with `npm` before I realized it was stuck on some DNS stuff. Sorry for the churn.

**Comment by lnicola** [comment]
Might be easier to revert the changes to `package-lock.json` and start over.

**Comment by davidbarsky** [comment]
> Might be easier to revert the changes to `package-lock.json` and start over.

yeah, already on that. are y'all okay with force pushes?

**Pull request merged**